thread/osx/priority.c

```diff
@@ -49,47 +45,43 @@ initialize_priority_map(void)
 	if (omrthread_lib_use_realtime_scheduling()) {
 		rc = initialize_realtime_priority_map();
 	} else {
+		const int default_priority[J9THREAD_PRIORITY_MAX + 1] = J9_PRIORITY_MAP;
 		uintptr_t i = 0;
-
-		/* Minimum, maximum, and default priorities are available from host_info().
-		 * The default is 31. pthread attributes must use this default to have the same
-		 * priority as the main thread.
-		 */
-		host_priority_info_data_t priorityInfoData;
-		mach_msg_type_number_t msgTypeNumber = HOST_PRIORITY_INFO_COUNT;
-		host_flavor_t flavor = HOST_PRIORITY_INFO;
-
-		if (KERN_SUCCESS == host_info(mach_host_self(), flavor, (host_info_t)&priorityInfoData, &msgTypeNumber)) {
-			for (i = 0; i <= J9THREAD_PRIORITY_MAX; i+=1) {
-				priority_map[i] = priorityInfoData.user_priority;
-			}
-		} else {
-			rc = -1;
+		for (i = 0; i <= J9THREAD_PRIORITY_MAX; i++) {
+			priority_map[i] = default_priority[i];
 		}
 	}
 	return rc;
 }
 
+static void initialize_thread_policies(int *policy_regular_thread, int *policy_realtime_thread);
+static void initializePolicy(int range_start, int range_end, int policy, int map[]);
+static void initializeRange(int start_index, int end_index, int range_min, int range_max, int values[]);
+static int getCurrentThreadParams(int *prio, int *policy);
+
+/* regular priorities and policies: those available to java.lang.Threads,
+ */
+
 static int policy_regular_thread;
 static int policy_realtime_thread;
 
-/* Represents the O/S limits for realtime priorities */
+/* represents the O/S limits for realtime priorities */
```

Comment:
Please avoid mixing these kinds of stylistic changes with functional changes as they add "noise" and making review changes more difficult.